pubspec.yaml

```diff
@@ -23,6 +23,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  image_picker: ^0.6.7+7
```

Comment:
Duplicate key

Reply:
Remove the duplicates from the 'pubspec.yaml' file and run `flutter pub get` to check if it works